src/button/exports.js

```
@@ -62,7 +66,19 @@ export function setupExports({ props, isEnabled } : ExportsProps)  {
                     return onApprove(data, actions);
                 },
                 onCancel,
-                onError
+                onError,
+                // eslint-disable-next-line no-shadow
+                upgradeFacilitatorAccessToken: (facilitatorAccessToken, orderID) => {
+                    const buyerAccessToken = getBuyerAccessToken() || '';
```

If buyer access token is not present we should throw an error here. Good for caller to know that the operation couldn't go ahead.

---

(plus a CAL log would be helpful too so we can track on our side)